run/CESM/HISTORY.rc

```diff
@@ -104,32 +97,6 @@ COLLECTIONS: 'Restart',
                               'OHwgtByAirMassColumnFull      ', 'GIGCchem',
```

Comment:
Do we need the 'GIGCchem' for CESM?  If so we can leave it.  We have since remeoved that from GEOS-Chem Classic HISTORY.rc templates.

Reply:
Now removed.